### PR TITLE
[ADD] Dockerfile + [FIX] sha256 by using crypto-js

### PR DESCRIPTION
**Pull request by sohelzerdoumi**
I Dockerize this project and fix sha256 issue because the project is on Google Archive now

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM php:5.6-apache
+
+# Install requirement
+RUN apt-get update
```
Comment (Tuxity) [inline Dockerfile]:
````
RUN apt-get update && apt-get install -y \
    nodejs \
    npm \
    git \
    curl \
    zip
```

It's better like that, to minimize the layers

**Comment by Tuxity** [comment]
Really good work ! Thank you

But I'm not a big fan of apache, Nginx is a loooooot better. It run really faster, and if we dockerize the app I would like with nginx :p

**Comment by sohelzerdoumi** [comment]
no problem, i will do it!

**Comment by Tuxity** [comment]
Oh another thing.

I think it would be better to add travis-ci (or something like that), into de the project to run the `composer install` and `bower install`. But I'm more familiar with Drone-ci

The goal is to prepare everything before put it in a container to build the image. And you will be able to remove multiple lines in your Dockerfile

**Comment by sohelzerdoumi** [comment]
i agree with you for your EDIT. But i don't know how drone-ci will decrease size of Dockerfile?

**Comment by sohelzerdoumi** [comment]
Are we ok with this?

**Comment by Tuxity** [comment]
No no it won't decrease de size, it's a better repartition of tasks if we can say, and less stuff in dockerfile to keep it simple as possible.

Hmm the nginx conf looks pretty good yeah 👍  I'm gonna test this PR later

**Comment by mpgn** [comment]
project is deprecated, check https://github.com/cakebox/cakebox/issues/216

**Pull request closed without merging**